docs/templates.md

```diff
@@ -401,6 +401,15 @@ This type cannot be printed. The following methods are defined.
   `"git-submodule"`, or `"conflict"`.
 * `.executable() -> Boolean`: True if the entry is an executable file.
 
+### AnnotationLine type
+
+The following methods are defined.
+
+* `.line_number() -> Integer`: 1-based line number.
+* `.commit() -> Commit`: Commit responsible for changing the relevant line.
+* `.first_line_in_hunk() -> Boolean`: False when the directly preceding line
```

Comment:
nit: I would reorder them as `commit, line_number, first_line_in_hunk` because `commit` is probably the most important, and methods are defined in that order.